spyder/plugins/editor/widgets/window.py

```diff
@@ -17,38 +17,95 @@
 import sys
 
 # Third party imports
+import qstylizer.style
 from qtpy.QtCore import QByteArray, QEvent, QPoint, QSize, Qt, Signal, Slot
 from qtpy.QtGui import QFont
 from qtpy.QtWidgets import (QAction, QApplication, QMainWindow, QSplitter,
                             QVBoxLayout, QWidget)
 
 # Local imports
-from spyder.api.config.mixins import SpyderConfigurationAccessor
+from spyder.api.plugins import Plugins
+from spyder.api.config.decorators import on_conf_change
+from spyder.api.config.mixins import SpyderConfigurationObserver
+from spyder.api.widgets.toolbars import ApplicationToolbar
+from spyder.api.widgets.mixins import SpyderToolbarMixin, SpyderWidgetMixin
 from spyder.config.base import _
 from spyder.plugins.editor.widgets.splitter import EditorSplitter
 from spyder.plugins.editor.widgets.status import (CursorPositionStatus,
                                                   EncodingStatus, EOLStatus,
                                                   ReadWriteStatus, VCSStatus)
+from spyder.plugins.mainmenu.api import (
+    ApplicationMenu,
+    ApplicationMenus,
+    MENUBAR_STYLESHEET,
+)
 from spyder.plugins.outlineexplorer.main_widget import OutlineExplorerWidget
-from spyder.py3compat import qbytearray_to_str, to_text_string
-from spyder.utils.icon_manager import ima
-from spyder.utils.qthelpers import (add_actions, create_action,
-                                    create_toolbutton)
-from spyder.utils.stylesheet import APP_STYLESHEET, APP_TOOLBAR_STYLESHEET
+from spyder.plugins.toolbar.api import ApplicationToolbars
+from spyder.py3compat import qbytearray_to_str
+from spyder.utils.palette import QStylePalette
+from spyder.utils.qthelpers import create_toolbutton
+from spyder.utils.stylesheet import APP_STYLESHEET
 from spyder.widgets.findreplace import FindReplace
 
 
 logger = logging.getLogger(__name__)
 
 
-class EditorWidget(QSplitter):
+# ---- Constants
+# -----------------------------------------------------------------------------
+class EditorMainWindowMenus:
+    View = "view"
+
+
+class ViewMenuSections:
+    Outline = "outline"
+    Toolbars = "toolbars"
+
+
+class EditorMainWindowActions:
+    ToggleOutline = "toggle_outline"
+
+
+# ---- Widgets
+# -----------------------------------------------------------------------------
+class OutlineExplorerinEditorWindow(OutlineExplorerWidget):
```

Comment:
Consistency in CamelCase capitalization.

```suggestion
class OutlineExplorerInEditorWindow(OutlineExplorerWidget):
```